core/src/main/java/it/pagopa/selfcare/mscore/core/NotificationServiceImpl.java

```diff
@@ -101,7 +107,7 @@ public void sendMailForApprove(User user, OnboardingRequest request, String toke
     }
 
     public void sendCompletedEmail(List<User> managers, Institution institution, Product product, File logo) {
-        List<String> destinationMails = new ArrayList<>(getCompleteDestinationMails(institution));
+        List<String> destinationMails = new ArrayList<>(getDestinationMails(institution));
```

Comment:
you might avoid to create a new ArrayList<>.......destinationMails = getDestinationMails(institution);